File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/playback/file/exoplayer/error/ExoPlayerException.java
package com.lasthopesoftware.bluewater.client.playback.file.exoplayer.error;

import com.lasthopesoftware.bluewater.client.playback.file.error.PlaybackException;
import com.lasthopesoftware.bluewater.client.playback.file.exoplayer.ExoPlayerPlaybackHandler;


public class ExoPlayerException extends PlaybackException {
	public ExoPlayerException(ExoPlayerPlaybackHandler playbackHandler) {
		super(playbackHandler);
	}

	public ExoPlayerException(ExoPlayerPlaybackHandler playbackHandler, Throwable cause) {
		super(playbackHandler, cause);
	}
}

File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/playback/file/exoplayer/error/ExoPlayerException.kt
package com.lasthopesoftware.bluewater.client.playback.file.exoplayer.error

import com.lasthopesoftware.bluewater.client.playback.file.error.PlaybackException
import com.lasthopesoftware.bluewater.client.playback.file.exoplayer.ExoPlayerPlaybackHandler

class ExoPlayerException : PlaybackException {
    constructor(playbackHandler: ExoPlayerPlaybackHandler) : super(playbackHandler) {}
    constructor(playbackHandler: ExoPlayerPlaybackHandler, cause: Throwable?) : super(
        playbackHandler,
        cause
    ) {
    }
}
